pf2/figures/figureA9.py

```diff
@@ -73,16 +79,16 @@ def plsr_acc_proba(patient_factor_matrix, meta_data, n_components=2, roc_auc=Tru
         score = roc_auc_score
     else:
         score = accuracy_score
-
+        
     covid_acc = score(
```

Comment:
In case we get any headaches from typing later, I'd recommend either using the `.to_numpy()` for all of the `probabilities` variables or none of them.